scripts/contrib/completion/bash/netctl

```diff
@@ -1,11 +1,19 @@
-#! /bin/bash
+#!/bin/bash
 
 GLOBAL_COMMANDS="\
+    version\
     group\
+    endpoint\
+    netprofile\
     network\
     tenant\
     policy\
+    external-contracts\
     global\
+    aci-gw\
+    bgp\
+    app-profile\
+    service\
     help"
```

Comment:
The completion script automatically sorts the completions when it displays them, the same as your above fix for the help.  So unless we re-order commands.go, it seems to make sense to leave this script in the original matching order.  All the bash functions in the script are also laid out the original order, so ideally we would then need to alphabetize the entire file.

```text
[vagrant@netplugin-node1 netplugin]$ netctl 
aci-gw              external-contracts  netprofile          tenant
app-profile         global              network             version
bgp                 group               policy              
endpoint            help                service             
```